modules/core/src/v2/coins/trx.ts

```diff
@@ -39,6 +42,16 @@ export interface TransactionPrebuild {
   };
 }
 
+export interface ExplainTransactionOptions {
+  txHex?: string;
+  transaction?: {
```

Comment:
Half signed and fully signed transactions have different shapes:
```
export interface FullySignedTransaction {
  txHex: string;
}

export interface HalfSignedTransaction {
  halfSigned?: {
    txHex?: string;
    payload?: string;
    txBase64?: string;
  };
}
```

Can this really be both? If so, we can use those base interfaces and say `transaction: HalfSignedTransaction | FullySignedTransaction`